### ISSUE TITLE
Pharyngalgia 

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#6061]
**Preferred term label:**
Pharyngalgia 

**Definition (free text, please give PubMed ID)**
Pain in the pharynx.
https://en.wiktionary.org/wiki/pharyngalgia#:~:text=Noun,Pain%20in%20the%20pharynx.

**Parent term (use hpo.jax.org/app)**
'Abnormality of the pharynx'

This is a symptom of COVID-19.

**Comment by nicolevasilevsky** [comment]
Or do you think this would be better for Mondo?

I don't see it in Mondo, but a similar term, Neuralgia (nerve pain), is in Mondo

**Comment by pnrobinson** [comment]
Neuralgia is more of a phenotypic feature than a disease!
It probably should not be a Mondo term.

**Comment by nicolevasilevsky** [comment]
ah, could we add Neuralgia as a class in HPO then? :)

**Comment by pnrobinson** [comment]
added as HP:0033154
We should addd Neuralgia as an HPO term
